internal/distroregistry/distroregistry_test.go

```diff
@@ -19,3 +21,16 @@ func TestRegistry_List(t *testing.T) {
 
 	require.ElementsMatch(t, expected, distros.List(), "unexpected list of distros")
 }
+
+func TestRegistry_GetDistro(t *testing.T) {
+	distros := NewDefault()
+
+	t.Run("distro exists", func(t *testing.T) {
+		expectedDistro := rhel8.New()
+		require.Equal(t, expectedDistro.Name(), distros.GetDistro(expectedDistro.Name()).Name())
+	})
+
+	t.Run("distro doesn't exist", func(t *testing.T) {
+		require.Nil(t, distros.GetDistro("toucan-os"))
```

Comment:
What's the reason we don't support Toucan OS?

Reply:
Because toucans can fly so they would make Tux jealous.

Reply:
Not true! https://youtu.be/9dfWzp7rYR4

Reply:
LOL, I love everything about the video. :D:D:D